redwood-layout-uiview/src/commonMain/kotlin/app/cash/redwood/layout/uiview/UIViewBox.kt

```diff
@@ -41,21 +42,25 @@ import platform.UIKit.UIEdgeInsetsMake
 import platform.UIKit.UIView
 import platform.darwin.NSInteger
 
-internal class UIViewBox : Box<UIView> {
+internal class UIViewBox :
+  Box<UIView>,
+  ResizableWidget<UIView> {
   override val value: View = View()
 
   override var modifier: Modifier = Modifier
 
   override val children get() = value.children
 
+  override var sizeListener: ResizableWidget.SizeListener? by value::sizeListener
```

Comment:
This used to produce wildly inefficient garbage code, but at some point it no longer does. Nice!

https://kotlin.godbolt.org/z/5heMY7KMr